src/style.css

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  --header-color-active: #00acdc;
+  --height-header: 60px;
+  --margin-header: 50px;
+  --bg-color-header: #fff;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--bg-color-header);
+  margin: 0 auto;
+}
+
+.header__nav {
+  margin-right: var(--margin-header);
+}
+
+.header__icon {
+  display: flex;
+  align-items: center;
+  margin-left: var(--margin-header);
+}
+
+.header__icon-link {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style: none;
+  margin-right: 20px;
+  position: relative;
+}
+
+.nav__item:last-child {
+  margin: 0;
+}
+
+.nav__item-link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  height: var(--height-header);
+  text-transform: uppercase;
+}
+
+.nav__item-link:hover {
+  color: var(--header-color-active);
+}
+
+.is-active {
+  color: var(--header-color-active);
+
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  background-color: var(--header-color-active);
+  height: 4px;
+  border-radius: 8px;
```

Comment:
border-radius should be half of width/height. If you set bigger value it is not a good practice

Reply:
you are right and I corrected it, it was according to figma hight: 4px, border: 8px, Thanks